ui/src/app/views/account-creation/component.html

```diff
@@ -29,10 +29,12 @@ <h5 style="display: inline-block;">@{{gsuiteDomain}}</h5>
     </div>
 
     <div class="form-section">
-      <button (click)="createAccount()" class="btn btn-primary short-button">Next</button>
-      <span class="spinner spinner-inline" *ngIf="creatingAcccount">
-        Creating account...
-      </span>
+      <button type="submit"
+          class="btn btn-primary short-button"
+          [disabled]="creatingAcccount">
+        Next
+      </button>
+      <span class="spinner spinner-inline" *ngIf="creatingAcccount"></span>
```

Comment:
I'd put `[clrLoading]="creatingAccount"` on the `<button>`... not sure how well that works anymore but it usually looks pretty clean.  You can [see it in action](https://vmware.github.io/clarity/documentation/v0.11/buttons) (bottom of the page).

Reply:
Good call - done.